### DISCUSSION TITLE
Network access

### DISCUSSION THREAD

**Discussion by pinghau** [DataDog/stratus-red-team#544] [Q&A] [answered]
Hi all,

I am trying to run stratus. I am able to run successfully with all my ports open up to internet. 

Was just wondering, will the stratus tool still work if i were to block off network access and run it locally?
Example:
Download stratus locally. 
Upload to S3.
Download via AWS Linux from my S3 bucket.
Then block off network.
Install stratus and set it up.
Then start the necessary attack.

**Comment by christophetd** [comment, marked as answer]
Hi there,

Since Stratus uses the cloud provider APIs, it requires network access. It should work fine if you're e.g. on an EC2 instance with all the VPC endpoints properly set up, if that's what you're asking, but I did not test that. 

Hope that helps!
